deployability/modules/allocation/README.MD

```diff
@@ -350,7 +350,7 @@ python3 modules/allocation/main.py --action create --provider vagrant --size lar
 
 #### General-specific functions
 
-- **Launcher** ([/wazuh-qa/deployability/modules/allocation/main.py](main.py)): The entry point for the workflow or the user who wishes to execute a test.
+- **Launcher** ([/wazuh-qa/deployability/modules/allocation/main.py](main.py)): The entry point for the jobflow or the user who wishes to execute a test.
```

Comment:
I don't think jobflow is the correct word there

Reply:
Fixed

https://github.com/wazuh/wazuh-qa/blob/746eb38dfeea6a992b7d4c4512a0dcae27561ea4/deployability/modules/allocation/README.MD?plain=1#L353